dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassDumperTest.java

```diff
@@ -16,7 +16,7 @@
 
 package com.google.cloud.tools.opensource.classpath;
 
-import static com.google.cloud.tools.opensource.classpath.LinkageCheckerTest.resolvePaths;
+import static com.google.cloud.tools.opensource.classpath.LinkageCheckerTest.resolve;
```

Comment:
This shouldn't be a static import, and probably one test should not be depending on another like this.